src/platform/CMakeLists.txt

```diff
@@ -23,9 +23,21 @@ if("${CX_PLATFORM}" STREQUAL "windows")
         set(SOURCES ${SOURCES} datapath_raw_win.c datapath_raw_socket.c datapath_raw_socket_win.c datapath_raw_xdp_win.c)
     endif()
 else()
+    set(LINUX_XDP_ENABLED FALSE)
     set(SOURCES ${SOURCES} inline.c platform_posix.c storage_posix.c cgroup.c)
     if(CX_PLATFORM STREQUAL "linux" AND NOT CMAKE_SYSTEM_NAME STREQUAL "FreeBSD")
-        set(SOURCES ${SOURCES} datapath_raw_linux.c datapath_raw_socket.c datapath_raw_socket_linux.c datapath_raw_xdp_linux.c datapath_epoll.c datapath_linux.c)
+        set(SOURCES ${SOURCES} datapath_linux.c datapath_epoll.c)
+        if ((${UBUNTU_VERSION} STREQUAL "20.04") OR
+            (${CMAKE_TARGET_ARCHITECTURE} STREQUAL "arm64") OR
+            (${CMAKE_TARGET_ARCHITECTURE} STREQUAL "arm") OR
+            (${CMAKE_SYSTEM_PROCESSOR} STREQUAL "arm") OR
```

Comment:
Interesting. XDP not supported outside of x64?

Reply:
I just wanted to start from small to get it merge.

Reply:
And realized that there are no libxdp/libbpf package provided for arm